packages/gatsby-source-mongodb/src/gatsby-node.js

```diff
@@ -45,25 +45,31 @@ exports.sourceNodes = (
             createNode,
             createNodeId,
             col,
-            createContentDigest
+            createContentDigest,
+            getNode,
+            deleteNode
           )
         )
       )
-        .then(() => {
-          mongoClient.close()
-        })
-        .catch(err => {
-          console.warn(err)
-          mongoClient.close()
-          return err
-        })
+      // .then(() => {
```

Comment:
Remove comments :)

Reply:
Still unsure if we need to wait on signal exit and gracefully end connection. In either case, connection should be kept open only if 

> in develop with mongodb 4.2 or greater, and you need a replica set.